cartridges/cloudinary_sg_changes/cartridge/templates/default/product/components/productimages.isml

```diff
@@ -82,12 +80,13 @@
                 <li class="thumb${imgSmStatus.first ? ' selected' : ''}">
                     <a ${href} class="thumbnail-link">
                         <img
-                            class="productthumbnail ${pdict.cloudinary && pdict.cloudinary.isEnabled ? 'cld-responsive' : ''}"
-                            src="${img.url}"
+                            class="productthumbnail ${pdict.cloudinary && pdict.cloudinary.isEnabled && img.isResponsive ? 'cld-responsive' : ''}"
                             alt="${img.alt}"
                             data-lgimg='{"url":"${img.url}", "title":"<isprint value="${lgImage.title}" encoding="on"/>", "alt":"<isprint value="${lgImage.alt}" encoding="on"/>", "hires":"${hi_res}", "prodID": "${pdict.Product.ID}", "cldEnabled": "${(pdict.cloudinary && pdict.cloudinary.isEnabled)}"}'
-                            <isif condition="${pdict.cloudinary && pdict.cloudinary.isEnabled}">
+                            <isif condition="${pdict.cloudinary && pdict.cloudinary.isEnabled && img.isResponsive}">
                                 data-src="${img.url}"
+                            <iselse/>
+                                src="${img.url}"
```

Comment:
where is the 'srcset' and 'sizes' attributes?

Reply:
It's not there, from the start. If you really need we can add that.